### PR TITLE
fix ansible remediation for zipl_bootmap_is_up_to_date

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- fix missing is defined in when clause

#### Rationale:

playbook for ospp was failing

- Fixes # 5926

### CODE REVIEW

**Comment by mildas** [comment]
**Changes identified:**
Rule zipl_bootmap_is_up_to_date:
 Ansible remediation changed.

**Recommended tests to execute:**
 build_product rhel8
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel8-ds.xml zipl_bootmap_is_up_to_date

File: linux_os/guide/system/bootloader-zipl/zipl_bootmap_is_up_to_date/ansible/shared.yml [hunk 1]
```diff
@@ -21,4 +21,4 @@
     - name: "Update zIPL bootmap"
       command: /usr/sbin/zipl
       changed_when: True
-      when: boot_bootmap.stat.mtime < zipl_conf.stat.mtime
+      when: boot_bootmap.stat.mtime is defined and  boot_bootmap.stat.mtime < zipl_conf.stat.mtime
```
Comment (yuumasato) [inline linux_os/guide/system/bootloader-zipl/zipl_bootmap_is_up_to_date/ansible/shared.yml:24]: Does it make sense to also add `is defined` to `zipl_conf.stat.mtime`?

**Comment by mildas** [comment]
**Changes identified:**
Rule zipl_bootmap_is_up_to_date:
 Ansible remediation changed.

**Recommended tests to execute:**
 build_product rhel8
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel8-ds.xml zipl_bootmap_is_up_to_date

**Comment by vojtapolasek** [comment]
Thank you for the catch, I fixed it.

**Pull request merged**